app/views/dashboard/_comment_moderate.html.erb

```diff
@@ -0,0 +1,11 @@
+<% if comment.status == 4 %>
+  <p class="alert alert-warning moderated">
+    <% if logged_in_as(['admin','moderator']) %>
+      <%= t('dashboard._comment_moderate.first_time_post') %>
+      <a class="btn btn-default btn-xs" href="/admin/publish_comment/<%= comment.id %>"><%= t('dashboard.moderate.approve') %></a>
+      <a class="btn btn-default btn-xs" href="/admin/mark_spam/<%= comment.id %>"><%= t('dashboard.moderate.spam') %></a>
```

Comment:
@ValentinaTironi this should be `mark_comment_spam` as `mark_spam` is used for marking node as spam.

Reply:
Got it! I'm gonna change it right now!

Reply:
Also, you could test your work at https://unstable.publiclab.org . Thanks!